.github/workflows/ci-docs.yml

```diff
@@ -0,0 +1,32 @@
+name: Documentation
+on:
+  pull_request:
+    branches: [main]
+  push:
+    # Runs on pushes targeting the default branch
+    branches: [main]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: prefix-dev/setup-pixi@v0.7.0
+      - run: pixi run docnb
+      - uses: actions/upload-pages-artifact@v3
+        with:
+          path: ./_build/html
```

Comment:
This has to match the output path in the `doc` task definition under `pixi.toml`.